neo/neo.csproj

```diff
@@ -23,6 +23,7 @@
   <ItemGroup>
     <PackageReference Include="Akka" Version="1.3.11" />
     <PackageReference Include="K4os.Compression.LZ4" Version="1.1.3" />
+    <PackageReference Include="LevelDB.Standard" Version="2.1.5" />
```

Comment:
I am scared about this, i love nugets, but this is a very humble nugget, so how could we are sure that the owner don't cheat us?

Reply:
@erikzhang , please please take a consideration about this.

Reply:
@shargon  can't we "have our own"? Like "neo leveldb" in nuget?

Reply:
This is a problem.

We also have a leveldb package on nuget, but it does not currently support .NET Core.

https://www.nuget.org/packages/LevelDB.Net/

Maybe we should modify this package and use it?

Reply:
I prefer to fork it and modify it. Then will be owned by neo.

Reply:
I was able to compile leveldb from google's repo and deploy it to work with neo-cli locally. So wouldn't it make more sense to start with the google's source than any other fork of the leveldb code?